docs/make.jl

```diff
@@ -1,4 +1,5 @@
-using Documenter, DiffEqFlux
+push!(LOAD_PATH, "../src/")
```

Comment:
```suggestion
```

Reply:
I already added it.

Reply:
I also added DocStringExtensions to have automatic generation of parts of the docstring. I won't use until I can resolve the current situation.